### PR TITLE
Ruby on Rails capstone project - Budget app

### PR DESCRIPTION
**Pull request by Gmanlove**
### **Highlights 🌟**

- Splash Screen is created.

- The homepage is created and all implementations are complete.

- The transactions page is complete.

- The 'Add Category' and 'Add Transaction' forms are implemented.

- All unit and integration tests are written.

### CODE REVIEW

File: app/views/expenses/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<%= render partial: 'shared/header',locals: {left:'back', path:root_path ,middle: 'EXPENSES $'+ number_with_precision(@expenses_sum, precision: 2).to_s }%>
```
Comment (franclobo) [inline app/views/expenses/index.html.erb:1]: - [ ] When I tried to back to the previous page after adding expenses an error occurred, please fix this issue for a better user experience 👇🏽 

image

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @Gmanlove, 

# ♻️ REQUIRED CHANGES ♻️ 

so-close-jim-halpert

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Highlights

- Use devise, cancancan, psql ✔️ 
- Good UI and UX ✔️ 
- All tests passed ✔️ 
- Professional documentation ✔️ 

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: app/views/expenses/index.html.erb (hunk 1 above)
Comment (Gmanlove) [reply app/views/expenses/index.html.erb:1]: Hello @franclobo, please the app is working perfectly well on my own end both online and local machine..... The error might be due network failure from your end. thanks

**Review by codecaiine — APPROVED** [review]
Hi @Gmanlove , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Good UI DESIGN ✔️ 
- Good project implementation ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**